### DISCUSSION TITLE
Change channel the bot sends messages in?

### DISCUSSION THREAD

**Discussion by Rhendak** [FKLC/WhatsAppToDiscord#195] [Q&A]
Hello all, 

I was able to succesfully set up the bot. It's now pumping out all the messages from a whatsapp group in a channel under the category named "whatsapp". The channel name is a random string of (phone)numbers (possibbly due it to being a whatsapp community group chat?). 

However, I would like for the bot to post the messages to a different channel, under a different section. I would like it to go to a category named "text channels" and the channel named "chat". How can I achieve for the bot to share the messages coming from whatsapp in that channel?

**Comment by theassemblerguy** [comment]
The easiest way to accomplish what you want is probably to just rename that channel and move it into your desired category. You can rename and move them as you like.

**Reply by Rhendak** [reply]
Thanks for your quick reply. 

I was able to move the messages to a different channel by changing the channel for the webhook the bot created, that was listed on the channel. 

However, now it seems that after every few messages or so, the bot crashes and closes down. 
I was able to find this in the error log, but it doesn't explain the problem to me, maybe someone else understands what is happening? 

> {"level":30,"time":1721841038748,"pid":1200,"hostname":"Desktop","version":"v0.10.26","browser":["Firefox (Linux)","",""],"helloMsg":{"clientHello":{"ephemeral":"NJvpljeYXZJPqH18yRn2QBxhcqOy542pADi8aW9JYQI="}},"msg":"connected to WA"}
{"level":30,"time":1721841038862,"pid":1200,"hostname":"Desktop","version":"v0.10.26","node":{"username":"1234567890","passive":true,"userAgent":{"platform":"MACOS","appVersion":{"primary":2,"secondary":2413,"tertiary":1},"mcc":"000","mnc":"000","osVersion":"0.1","manufacturer":"","device":"Desktop","osBuildNumber":"0.1","releaseChannel":"RELEASE","localeLanguageIso6391":"en","localeCountryIso31661Alpha2":"US"},"connectType":"WIFI_UNKNOWN","connectReason":"USER_ACTIVATED","device":35},"msg":"logging in..."}
{"level":30,"time":1721841039174,"pid":1200,"hostname":"Desktop","version":"v0.10.26","msg":"23 pre-keys found on server"}
{"level":30,"time":1721841039284,"pid":1200,"hostname":"Desktop","version":"v0.10.26","msg":"opened connection to WA"}
{"level":30,"time":1721841039391,"pid":1200,"hostname":"Desktop","version":"v0.10.26","msg":"handled 24 offline messages/notifications"}
{"level":50,"time":1721841039419,"pid":1200,"hostname":"Desktop","version":"v0.10.26","err":{"type":"Error","message":"bad-request","stack":"Error: bad-request\n    at assertNodeErrorFree (C:\\snapshot\\WhatsAppToDiscord\\out.js)\n    at query (C:\\snapshot\\WhatsAppToDiscord\\out.js)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async fetchProps (C:\\snapshot\\WhatsAppToDiscord\\out.js)\n    at async Promise.all (index 1)\n    at async executeInitQueries (C:\\snapshot\\WhatsAppToDiscord\\out.js)","data":400,"isBoom":true,"isServer":true,"output":{"statusCode":500,"payload":{"statusCode":500,"error":"Internal Server Error","message":"An internal server error occurred"},"headers":{}}},"msg":"unexpected error in 'init queries'"}
{"level":50,"time":1721841140462,"pid":1200,"hostname":"Desktop","version":"v0.10.26","err":{"type":"DiscordAPIError","message":"Unknown Message","stack":"DiscordAPIError: Unknown Message\n    at RequestHandler.execute (C:\\snapshot\\WhatsAppToDiscord\\out.js)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async RequestHandler.push (C:\\snapshot\\WhatsAppToDiscord\\out.js)\n    at async MessageManager._fetchId (C:\\snapshot\\WhatsAppToDiscord\\out.js)\n    at async Client.<anonymous> (C:\\snapshot\\WhatsAppToDiscord\\out.js)","name":"DiscordAPIError","method":"get","path":"/channels/1265709310743412847/messages/1265718204173389958","code":10008,"httpStatus":404,"requestData":{"files":[]}},"msg":"Unknown Message"}
{"level":30,"time":1721841140462,"pid":1200,"hostname":"Desktop","version":"v0.10.26","msg":"Exiting!"}
